src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -169,25 +199,48 @@ public TimeSpan TimeToLive
         }
 
         /// <summary>Gets or sets the a correlation identifier.</summary>
-        /// <value>The identifier of the correlation.</value>
-        /// <remarks>Its a custom property that can be used to either transfer a correlation Id to the destination or be used in <see cref="CorrelationFilter"/></remarks>
+        /// <value>Correlation identifier.</value>
+        /// <remarks>
+        ///    Allows an application to specify a context for the message for the purposes of correlation, 
+        ///    for example reflecting the MessageId of a message that is being replied to.
+        ///    See <a href="https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messages-payloads?#message-routing-and-correlation">Message Routing and Correlation</a>.
+        /// </remarks>
         public string CorrelationId { get; set; }
 
-        /// <summary>Gets or sets the application specific label.</summary>
-        /// <value>The application specific label.</value>
+        /// <summary>Gets or sets an application specific label.</summary>
+        /// <value>The application specific label</value>
+        /// <remarks>
+        ///   This property enables the application to indicate the purpose of the message to the receiver in a standardized 
+        ///   fashion, similar to an email subject line. The mapped AMQP property is "subject".
```

Comment:
Would be nice to have a link to a document that describes these mappings.

Reply:
I'm adding a link here https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messages-payloads to the class description.

Reply:
👍 If may suggest to split `Property Name` column and `mapping property`